UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Services/BuilderService.cs

```diff
@@ -29,13 +32,17 @@ public class BuilderService : IBuilderService
         private readonly IAzureTableStorageHelper azureTableStorageHelper;
         private readonly IOptions<FssApiConfiguration> fssApiConfig;
         private readonly IPksService pksService;
+        private readonly IPermitDecryption permitDecryption;
 
         private const string BESPOKE_FILE_NAME = "V01X01";
         private const string BessBatchFileExtension = "zip";
+        private const string PermitTextFile = "Permit.txt";
+        private const string PermitXmlFile = "Permit.xml";
```

Comment:
Suggestion: can we decouple extension from filename?
```suggestion
        private const string PERMIT_FILE_NAME= "Permit";
```